packages/@react-aria/collections/src/CollectionView.tsx

```diff
@@ -70,11 +71,11 @@ export function CollectionView<T extends object, V>(props: CollectionViewProps<T
     }
 
     isFocusWithin.current = e.target !== ref.current;
-  }, [focusedKey, collectionManager]);
+  }, [ref, collectionManager, focusedKey]);
 
   let onBlur = useCallback((e: FocusEvent) => {
     isFocusWithin.current = ref.current.contains(e.relatedTarget as Element);
-  }, []);
+  }, [ref]);
```

Comment:
I think @LFDanLu figured this should be ref.current?

Reply:
Then we'd be making a new function every time the ref changed value. `ref` is mutated so I think this is correct as it is.